frontend/coprs_frontend/coprs/auth.py

```diff
@@ -22,38 +19,48 @@ class UserAuth:
     encapsulated within this class.
     """
 
-    @staticmethod
-    def logout():
+    @classmethod
+    def next_url(cls):
+        """
+        Where should user be redirected after a successful login?
```

Comment:
Ideally to the same page where he comes from (when clicking on the login button).